### DISCUSSION TITLE
这个绳网真的匿名吗？

### DISCUSSION THREAD

**Discussion by AnkaAlioth** [share121/inter-knot#2758] [其他]
<img width="920" alt="屏幕截图 2024-08-18 205248" src="https://github.com/user-attachments/assets/43aebd2f-3ea4-4f57-9f27-4f3a80afd8e7">

发委托不会被治安局查了吧<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by ekjjjjjjj** [comment]
v我50就告诉你

**Reply by AnkaAlioth** [reply]
V你50丁尼，怎么收账

**Reply by ekjjjjjjj** [reply]
零号空洞见

**Reply by LeiMoo** [reply]
> 零号空洞见

你不会黑吃黑吧，我们用绳网交易吧/xyx
![image](https://github.com/user-attachments/assets/91bdd748-ad8a-4c2f-ad44-2d3a9cb45e8b)

**Comment by LeiMoo** [comment]
朱鸢长官也会访问的， 只要你言论符合这些
![image](https://github.com/user-attachments/assets/bacb9378-b530-480a-ba98-e3819d3cc10e)
基本不会敲门查水表

**Comment by Kujojolen** [comment]
我们绳匠的眼睛是雪亮的，不合理的行为会被举办给治安局哦～

**Comment by Shuyluo** [comment]
你好，请问是网上做虚拟歌手的初音未来同学吗？

**Reply by AnkaAlioth** [reply]
XXX，有盒！

**Reply by LeiMoo** [reply]
> XXX，有盒！

只要你不主动泄露，只能通过邮箱查询你信息，届时就看你邮箱的问题了

**Reply by MaskedFools-Mochun** [reply]
XXX，有盒！
